### ISSUE TITLE
[input] When editing a cell, Ctrl-O stack traces

### ISSUE THREAD

**Issue by frosencrantz** [saulpw/visidata#1116]
**Small description**
When editing a cell, Ctrl-O stack traces, wrong number of args

**Expected result**
Opens cell value in the editor
**Actual result with screenshot**
Got up a little early to send this in, but don't have a screenshot.

**Steps to reproduce with sample data and a .vd**
Open Visidata, start to edit a cell, then try Ctrl-O

The issue is on this line, the parameters have changed recently:
  https://github.com/saulpw/visidata/blob/develop/visidata/_input.py#L218

**Additional context**
Please include the version of VisiData. Latest version of develop

Also, congratulations on the new release! 🎉

**Comment by anjakefala** [comment]
Excellent catch @frosencrantz!

**Comment by anjakefala** [comment]
I fixed this on `develop`, and it will be in v2.6.1. I will add `sysopen-row` to the manual test checklist.

**Comment by frosencrantz** [comment]
Wonderful! Thank you @anjakefala !   That was fast.

I really ❤️ that you added it to the pre-release checklist!

One thing that might have helped catch this bug automatically would be a Python type checker like mypy.  My understanding is that type annotations can be added over time, and don't need to be done all at once.  The annotations would need to be added to get the benefits. 

It looks like Will uses mypy for Rich:  https://github.com/willmcgugan/rich/blob/master/.github/workflows/pythonpackage.yml#L34-L37

Something like that might help Visidata be even more robust.

**Comment by saulpw** [comment]
Hey @frosencrantz, thanks for the suggestion.  I've tried adopting Python type annotations in the past, and I find them too noisy for my own comfort.  For instance, to catch this particular bug, every `@VisiData.api` function would need its `vd` argument to become `vd:VisiData`, which is superfluous given the surrounding context. The final straw was that these annotations are completely ignored by the interpreter and builtin tools, and would require an external type checker like mypy as you mention.  I might try to use type annotations in a future project from the start, but for VisiData it would be a huge amount of effort, and not worth it from my perspective.  I'd rather create more robust testing strategies outside the library/tool, than clutter the codebase.  Readability is key!

(FWIW I do use extensive typing in low-level C code, to the point that when I've taught workshops on writing custom kernels, I tell people that they should have typedefs for everything--their code should never actually use `int`.  So I appreciate good type systems.  I've just found it more annoying than useful for small projects like VisiData.)
